atc/worker/worker.go

```diff
@@ -207,6 +212,64 @@ func (worker *gardenWorker) LookupVolume(logger lager.Logger, handle string) (Vo
 	return worker.volumeClient.LookupVolume(logger, handle)
 }
 
+func secureImageSource(delegate ImageFetchingDelegate, source atc.Source) atc.Source {
+	newSource, _ := delegate.ImageSourceRedaction(source)
```

Comment:
This error should probably not be dropped, even though it looks pretty unlikely to happen right now.

Reply:
Fixed.